src/sections/Sponsors.jsx

```diff
@@ -215,37 +73,19 @@ const Sponsors = () => {
 
   return (
     <SponsorsContainer>
-      <SponsorsBackground src={SponsorBackgroundImage} alt="Sponsor section background" />
-      <SponsorsBackgroundMobile src={SponsorBackgroundMobile} alt="Sponsor section background" />
-      {stamps.map((item, index) => (
-        <Stamp
-          ref={stampPArr[index].ref}
-          key={item.name}
-          src={item.imgSrc}
-          alt={item.name}
-          top={item.top || null}
-          right={item.right || null}
-          bottom={item.bottom || null}
-          left={item.left || null}
-        />
-      ))}
-      <DuckContainer>
-        <Duck src={DuckImage} alt="A cute large rubber ducky" />
-        <DuckRipples src={DuckRipplesImage} alt="Ripples in the water caused by a big rubber ducky" />
-      </DuckContainer>
-      <SeabusContainer>
-        <Seabus src={SeabusImage} alt="A seabus" />
-        <SeabusRipples src={SeabusRipplesImage} alt="Ripples in the water caused by a seabus" />
-      </SeabusContainer>
       <StaticContainer>
         <StyledTitle id="sponsors">Sponsors</StyledTitle>
         <PushinP>
           Sponsors make this event happen. If you are interested in working with us, joining us or speaking at one of
           our events, please reach out to us below!
         </PushinP>
-        <Button variant="solid" href="mailto:sponsorship@nwplus.io">
+        <Button variant="solidRed" href="mailto:sponsorship@nwplus.io">
           Sponsor nwHacks!
         </Button>
+        <Carousel>
+          <p>Hello</p>
```

Comment:
Could you link up the Carousel to dynamically fetch sponsors from the firebase collection?

![image](https://github.com/nwplus/external-templates/assets/62491197/38e426c3-4ae0-4fe8-8b45-4942bdb10a10)